src/ServiceControl.Monitoring/Licensing/LicenseManager.cs

```diff
@@ -12,11 +11,9 @@ public class LicenseManager
         public void Refresh()
         {
             Logger.Debug("Checking License Status");
-            var sources = new LicenseSource[]
-            {
-                new LicenseSourceFilePath(LicenseFileLocationResolver.GetPathFor(Environment.SpecialFolder.CommonApplicationData))
```

Comment:
As long as this location is included in the `GetStandardLicenseSources()` list ([which it seems to be](https://github.com/Particular/Operations.Licensing/blob/master/src/Particular.Licensing/FindActiveLicense/LicenseSource.cs#L65)) then I am happy.

Reply:
Yes it is, and I don't see us removing that any time soon.